.github/workflows/ci.yml

```diff
@@ -14,55 +14,55 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout Kibana
+      - name: Checkout OpenSearch Dashboard
         uses: actions/checkout@v2
         with:
-          path: kibana
-          repository: elastic/kibana
+          path: OpenSearch-Dashboards
+          repository: opensearch-project/OpenSearch-Dashboards
           fetch-depth: 0
       - name: Create plugins dir
         run: |
-          cd ./kibana
+          cd ./OpenSearch-Dashboards
           mkdir -p plugins
       - name: Checkout
         uses: actions/checkout@v2
         with:
-          path: kibana/plugins/opendistro_security
+          path: OpenSearch-Dashboards/plugins/security-dashboards-plugin
           ref: ${{ github.ref }}
-      - name: Get Kibana version
-        id: kbn_version
+      - name: Get OpenSearch Dashboards version
+        id: osd_version
         run: |
-          echo "::set-output name=kbn_version::$(jq -r '.kibana.version' ./kibana/plugins/opendistro_security/package.json)"
-      - name: Use kibana release commit
+          echo "::set-output name=osd_version::$(jq -r '.opensearchDashboards.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json)"
+      - name: Use OpenSearch release commit
         run: |
-          cd ./kibana
-          git checkout tags/v${{ steps.kbn_version.outputs.kbn_version }} -b v${{ steps.kbn_version.outputs.kbn_version }}
+          cd ./OpenSearch-Dashboards
+          # git checkout tags/v${{ steps.osd_version.outputs.osd_version }} -b v${{ steps.osd_version.outputs.osd_version }}
```

Comment:
Why is this commentted out?

Reply:
there is no tag `1.0.0-beta1` in OpenSearch repo, should be able to re-introduce it after things settle in it